flowise-embed/src/components/bubbles/SelectionBubble.tsx

```diff
@@ -0,0 +1,138 @@
+import { Show, onMount, createEffect, For, Setter } from 'solid-js';
+import { Avatar } from '../avatars/Avatar';
+import { Marked } from '@ts-stack/markdown';
+import { IAction, MessageType } from '../Bot';
+import { messageUtils } from '@/utils/messageUtils';
+import { colorTheme } from '@/utils/colorUtils';
+
+type Props = {
+  message: MessageType;
+  chatflowid: string;
+  chatId: string;
+  apiHost?: string;
+  fileAnnotations?: any;
+  showAvatar?: boolean;
+  avatarSrc?: string;
+  backgroundColor?: string;
+  textColor?: string;
+  chatFeedbackStatus?: boolean;
+  fontSize?: number;
+  feedbackColor?: string;
+  isLoading: boolean;
+  showAgentMessages?: boolean;
+  handleActionClick: (label: string, action: IAction | undefined | null) => void;
+  setMessages: Setter<MessageType[]>;
+  handleSubmit: (inputValue: string, action?: IAction | null) => void;
+  clearChat: () => void;
+  selectionOptions: [string, string];
```

Comment:
```suggestion
  selectionOptions: string[];
```
Não precisam ser apenas 2 opções, certo?